sql-plugin/src/main/scala/com/nvidia/spark/rapids/TypeChecks.scala

```diff
@@ -1506,6 +1506,21 @@ class CastChecks extends ExprChecks {
   }
 }
 
+/** 
+ * This class is just restricting the 'to' dataType to a StringType in the CastChecks class
+ */ 
+class ToPrettyStringChecks extends CastChecks {
+
+  override protected def tagBase(meta: RapidsMeta[_, _, _], willNotWork: String => Unit): Unit = {
+    val cast = meta.wrapped.asInstanceOf[UnaryExpression]
+    val from = cast.child.dataType
+    val to = StringType
+    if (!gpuCanCast(from, to)) {
```

Comment:
I don't think this is going to show up in the generated docs, even if we tried to build them with 3.5.0. We should at least have a follow on issue to figure that out.

Reply:
Wondering if there is a valid need to print `ToPrettyString` as part of the `supported_ops`. It is a sub set of what `GpuCast` can do.